src/client/pages/OfferNew/FaqSection.tsx

```diff
@@ -43,8 +43,7 @@ const TabsWrapper = styled.div`
 
 const StyledAccordionItem = styled(AccordionItem)`
   &:not(:last-child) {
-    margin-bottom: 1.5rem;
-
+    padding-bottom: 1.5rem;
     ${LARGE_SCREEN_MEDIA_QUERY} {
       margin-bottom: 2.5rem;
```

Comment:
Shouldn't this also be padding instead of margin?

Reply:
Yes, will add that! Thank you :)) Although for some reason it looked fine on bigger screens with the margin 🤔 But nice with consistency!